Scroller/EditableWrapper.js

```diff
@@ -44,16 +69,19 @@ const EditableWrapper = (props) => {
 
 	// Reset values
 	const reset = useCallback(() => {
-		const {selectedItem} = mutableRef.current;
+		const {selectedItem, spotlightId} = mutableRef.current;
 
-		selectedItem?.classList.remove(css.selected);
-		selectedItem?.classList.remove(css.rearranged);
+		selectedItem?.classList.remove(componentCss.selected);
+		selectedItem?.classList.remove(customCss.selected);
+		selectedItem?.classList.remove(componentCss.rearranged);
 
 		mutableRef.current.selectedItem = null;
 		mutableRef.current.lastMoveDirection = null;
 		mutableRef.current.prevToIndex = null;
 		wrapperRef.current.style.setProperty('--selected-item-offset', '0px');
-	}, []);
+
+		Spotlight.set(spotlightId, {restrict: 'self-first'});
+	}, [customCss.selected]);
```

Comment:
🤔 Which one is the right? `customCss` or this?

Reply:
Hm, When I set `customCss`, lint error occured
EditableWrapper.js
  39:6  warning  The 'customCss' logical expression could make the dependencies of useCallback Hook (at line 82) change on every render. To fix this, wrap the initialization of 'customCss' in its own useMemo() Hook  react-hooks/exhaustive-deps

Reply:
🆗